plugin/src/main/java/org/jenkinsci/plugins/casc/CasCGlobalConfig.java

```diff
@@ -0,0 +1,46 @@
+package org.jenkinsci.plugins.casc;
+
+import hudson.Extension;
+import jenkins.model.GlobalConfiguration;
+import net.sf.json.JSONObject;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.StaplerRequest;
+
+/**
+ *
+ */
+@Extension
+public class CasCGlobalConfig extends GlobalConfiguration {
+
+    private String configurationPath;
+
+    @DataBoundConstructor
```

Comment:
constructor won't be involved when using `bindJson(bean, json)`. Use a DataBoundSetter instead